presto-main/src/test/java/io/prestosql/security/TestFileBasedSystemAccessControl.java

```diff
@@ -268,16 +272,64 @@ public void testTableOperations()
+
+                    accessControlManager.checkCanCreateTable(bobContext, staffTable);
+                    accessControlManager.checkCanDropTable(bobContext, staffTable);
+                    accessControlManager.checkCanSelectFromColumns(bobContext, staffTable, ImmutableSet.of());
+                    accessControlManager.checkCanInsertIntoTable(bobContext, staffTable);
+                    accessControlManager.checkCanDeleteFromTable(bobContext, staffTable);
+                    accessControlManager.checkCanAddColumns(bobContext, staffTable);
+                    accessControlManager.checkCanRenameColumn(bobContext, staffTable);
+
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanCreateTable(nonAsciiContext, aliceTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanDropTable(nonAsciiContext, aliceTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanSelectFromColumns(nonAsciiContext, aliceTable, ImmutableSet.of()));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanInsertIntoTable(nonAsciiContext, aliceTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanDeleteFromTable(nonAsciiContext, aliceTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanAddColumns(nonAsciiContext, aliceTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanRenameColumn(nonAsciiContext, aliceTable));
+
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanCreateTable(nonAsciiContext, staffTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanDropTable(nonAsciiContext, staffTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanSelectFromColumns(nonAsciiContext, staffTable, ImmutableSet.of()));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanInsertIntoTable(nonAsciiContext, staffTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanDeleteFromTable(nonAsciiContext, staffTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanAddColumns(nonAsciiContext, staffTable));
+                    assertThrows(AccessDeniedException.class, () -> accessControlManager.checkCanRenameColumn(nonAsciiContext, staffTable));
                 });
         assertThrows(AccessDeniedException.class, () -> transaction(transactionManager, accessControlManager).execute(transactionId -> {
             accessControlManager.checkCanCreateTable(new SecurityContext(transactionId, bob), aliceTable);
```

Comment:
I think this check could move above

Reply:
It's actually a duplicate of one of my previous checks, I will remove it.